packages/yoga/.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../babel.config.js",
+  "presets": [
+    [
+      "@babel/preset-typescript"
+    ]
+  ]
+}
```

Comment:
Don't forget the empty line at EOF, Here why:

https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b